ui-tests/tests/general.test.ts

```diff
@@ -46,7 +46,7 @@ describe('General Tests', () => {
   });
 
   test('Toggle Light theme', async () => {
-    await galata.theme.setLightTheme();
+    await expect(galata.theme.setLightTheme()).resolves.toBeUndefined();
```

Comment:
Was this `expect` added because of the ESLint warning?

Wondering whether we could have something a bit more explicit for that particular test, for example using `galata.theme.getTheme`?

Reply:
> Was this expect added because of the ESLint warning?

indeed the eslint rule [jest expect-expect](https://github.com/jest-community/eslint-plugin-jest/blob/v24.1.3/docs/rules/expect-expect.md) raises a warning on this

> Wondering whether we could have something a bit more explicit for that particular test, for example using galata.theme.getTheme?

indeed this would be better - I'll update the PR

Reply:
:+1: